### DISCUSSION TITLE
Get Started with SenseCAP Indicator | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1557] [Q&A]
# Get Started with SenseCAP Indicator | Seeed Studio Wiki

Get Started with SenseCAP Indicator

https://wiki.seeedstudio.com/Sensor/SenseCAP/SenseCAP_Indicator/Get_started_with_SenseCAP_Indicator/

**Comment by Mapperdave** [comment]
What is the Refresh Rate of the SenseCAP Indicator 4-inch?

**Comment by PaulSchulz** [comment]
Can you please confirm that the Grove connector Pin-Out shown is correct for both Grove Connecotrs? The GND/VCC pins are shown to be different (at opposite ends) between the two connectors.

As far as I can tell, Grove(IIC) is correct,, and the Grove(ABC) has been mirrored in the above documentation.

**Reply by Jessie219-web** [reply]
thanks for your feedback, we will check it asap

**Reply by david9600** [reply]
I noticed this too (while figuring out how to attach a 3-wire temp/humidity sensor).
A voltmeter tells me that the outline of the ADC connector is rotated 180 degrees.  The IIC connector is shown correctly.

**Comment by danuw** [comment]
Hi, I was wondering:  Is there anyway to get the HA firmware in a format that can be dropped (without needing to build it locally?) - not sure I saw in the instruction that anything needed to be customised before being built and flashed apart from config in HA please?

**Reply by Zeke5579** [reply]
You may first need to design the HA features you need yourself and then flash the firmware. You can refer to the following article:https://wiki.seeedstudio.com/sensecap_indicator_project/#2-squareline-studio

https://www.hackster.io/spenyan/low-code-ui-design-for-sensecap-indicator-with-squareline-9825fe

**Comment by ril3y** [comment]
Where is the hardware schematic for this board? for the d1l? trying to write some custom (non meshtastic code) and cannot seem to find this anywhere?  swd? jtag pins etc? how the  cpus connect to eachother? anything?

**Reply by david9600** [reply]
Maybe the system diagram on p.13 of this document helps...
https://files.seeedstudio.com/products/SenseCAP/SenseCAP_Indicator/SenseCAP%20Indicator%20User%20Manual_2023.4.21.pdf
